books.js

```diff
@@ -0,0 +1,59 @@
+const titleInput = document.getElementById('title-input');
+const authorInput = document.getElementById('author-input');
+const submitButton = document.getElementById('submit-button');
+let books = [];
+
+function removeBook(book) {
+  books = books.filter((element) => element.title !== book.title);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function removeBtnEventListener(element, title, author) {
+  const book = { title, author };
+  removeBook(book);
+  element.parentElement.remove();
+}
+
+function addBookToDocument(book) {
+  const booksDiv = document.getElementById('books');
+  const bookDiv = document.createElement('div');
+  const title = document.createElement('p');
+  title.classList.add('title');
+  const author = document.createElement('p');
+  author.classList.add('author');
+  const remove = document.createElement('button');
+  remove.classList.add('remove-btn');
+
+  title.innerText = book.title;
+  author.innerText = book.author;
+  remove.innerText = 'Remove';
+
+  bookDiv.append(title, author, remove);
+  booksDiv.append(bookDiv);
+  remove.addEventListener('click', () => {
+    removeBtnEventListener(remove, book.title, book.author);
+  });
```

Comment:
- The localStorage is not working as expected on the remove or delete functionalities from the localStorage

- When a user sends the data, the input needs to get updated or become empty so that a user should send another data as well. 

![image](https://user-images.githubusercontent.com/55635977/152884938-c83c8ae1-b405-44c7-a620-d4fa4a3e9874.png)

Reply:
Can You Kindly clarify about localStorage problem?

Reply:
Hi @AlejandroNeira98, Adding to the localStorage is working perfectly however when you delete one data in the listed data the whole data gets deleted from the localStorage.

Reply:
Okay Thanks alot I understood your point.